### ISSUE TITLE
Running example project not working

### ISSUE THREAD

**Issue by FrenchMasta** [jsreport/jsreport-dotnet-example-webapp#6]
I have cloned and ran the example project (as is) but I cannot get it to work.

`FileNotFoundException: Could not find file 'C:\Users\ABJR696\AppData\Local\Temp\jsreport\autocleanup\metae463491a-3049-493b-af0c-51808e701aa2'.
System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
System.IO.FileStream..ctor(string path, FileMode mode, FileAccess access, FileShare share, int bufferSize, FileOptions options)
System.IO.StreamReader.ValidateArgsAndOpenPath(string path, Encoding encoding, int bufferSize)
System.IO.StreamReader..ctor(string path, Encoding encoding, bool detectEncodingFromByteOrderMarks)
System.IO.File.InternalReadAllText(string path, Encoding encoding)
System.IO.File.ReadAllText(string path)
jsreport.Local.Internal.LocalUtilityReportingService.RenderAsync(string requestString, CancellationToken ct)
jsreport.AspNetCore.JsReportMiddleware.Invoke(HttpContext context)
Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, object state, bool isCompleted)
Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(ref State next, ref Scope scope, ref object state, ref bool isCompleted)
Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, object state, bool isCompleted)
Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)`

![jsReport Error Snapshot](https://user-images.githubusercontent.com/25319525/92238693-0432fe80-eeba-11ea-96fd-1d3b5b3a497c.png)

I then hit the same endpoint again (Invoice) and get a different error:

![jsReport Error Snapshot 2](https://user-images.githubusercontent.com/25319525/92239140-caaec300-eeba-11ea-9269-bcca31cc5294.png)

**Comment by FrenchMasta** [comment]
Ran on a different laptop (not corporate laptop) and it worked. It may be an issue with access to file system or some sort of permission issue

**Comment by pofider** [comment]
I wasn't able to replicate this. Don't know what could be the reason, maybe antivirus blocking access to that file, or something deleted it.

Maybe it would help to set [temp directory](https://jsreport.net/learn/dotnet-local#temp-files) somewhere next to your project.

**Comment by FrenchMasta** [comment]
It seems setting `.KeepAlive(false)` fixes it.
I've tested this setting on both the **.NET Core Console** & **.NET Core Web App** projects. Both of which I configured to use a temp directory as well as use the original configuration.
I also tested (and found the problem) by running the executable via CMD

Setting `.KeepAlive(false)` on my own **.NET Framework project** (current version is v4.7.2) provides me with:
![image](https://user-images.githubusercontent.com/25319525/92610953-92dbbd00-f2b8-11ea-9d0c-a15b936552ec.png)

Since this issue was for the **.NET Core Web App** project, however, I will close this issue.
